### PR TITLE
add hostPorts to pod describer

### PR DESCRIPTION
**Pull request by dixudx**
**What this PR does / why we need it**:
Missing `HostPorts` when describing pods

**Which issue(s) this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close the issue(s) when PR gets merged)*:
Fixes #

**Special notes for your reviewer**:
/assign @mengqiy @shiywang
**Release note**:

```release-note
None
```

### CODE REVIEW

File: pkg/printers/internalversion/describe.go [hunk 1]
```diff
@@ -1321,6 +1321,14 @@ func describeContainerBasicInfo(container api.Container, status api.ContainerSta
 			w.Write(LEVEL_2, "Port:\t%s\n", portString)
 		}
 	}
+	hostPortString := describeContainerHostPorts(container.Ports)
+	if strings.Contains(hostPortString, ",") {
+		w.Write(LEVEL_2, "HostPorts:\t%s\n", hostPortString)
+	} else {
+		if len(hostPortString) == 1 {
```
Comment (php-coder) [inline pkg/printers/internalversion/describe.go]: This condition looks weird. When we the length of the string can be 1?

Comment (dixudx) [reply pkg/printers/internalversion/describe.go]: My mistake. Updated. PTAL. Thanks.

File: pkg/printers/internalversion/describe.go [hunk 2]
```diff
@@ -1321,6 +1321,16 @@ func describeContainerBasicInfo(container api.Container, status api.ContainerSta
...
 		}
 	}
+	hostPortString := describeContainerHostPorts(container.Ports)
+	if strings.Contains(hostPortString, ",") {
+		w.Write(LEVEL_2, "HostPorts:\t%s\n", hostPortString)
+	} else {
+		if len(hostPortString) == 0 {
+			w.Write(LEVEL_2, "HostPort:\t<none>\n")
+		} else {
+			w.Write(LEVEL_2, "HostPort:\t%s\n", hostPortString)
```
Comment (php-coder) [inline pkg/printers/internalversion/describe.go]: I think that having more `if` conditions just for respecting plural form doesn't worth it. This code also could benefit from using `stringOrNone()`:
```golang
w.Write(LEVEL_2, "HostPorts:\t%s\n", stringOrNone(hostPortString))
```

Comment (dixudx) [reply pkg/printers/internalversion/describe.go]: @php-coder Excellent.

Updated. PTAL. Thanks.

**Comment by dixudx** [comment]
@kubernetes/sig-cli-pr-reviews PTAL. Thanks.

File: pkg/printers/internalversion/describe.go [hunk 3]
```diff
@@ -1315,11 +1315,13 @@ func describeContainerBasicInfo(container api.Container, status api.ContainerSta
...
-		if len(portString) == 0 {
-			w.Write(LEVEL_2, "Port:\t<none>\n")
-		} else {
-			w.Write(LEVEL_2, "Port:\t%s\n", portString)
-		}
+		w.Write(LEVEL_2, "Port:\t%s\n", stringOrNone(portString))
+	}
+	hostPortString := describeContainerHostPorts(container.Ports)
+	if strings.Contains(hostPortString, ",") {
+		w.Write(LEVEL_2, "HostPorts:\t%s\n", hostPortString)
```
Comment (juanvallejo) [inline pkg/printers/internalversion/describe.go]: nit: s/HostPorts/Host Ports/

File: pkg/printers/internalversion/describe.go [hunk 4]
```diff
@@ -1315,11 +1315,13 @@ func describeContainerBasicInfo(container api.Container, status api.ContainerSta
...
+	} else {
+		w.Write(LEVEL_2, "HostPort:\t%s\n", stringOrNone(hostPortString))
```
Comment (juanvallejo) [inline pkg/printers/internalversion/describe.go]: nit: s/HostPort/Host Port/

**Comment by juanvallejo** [comment]
@dixudx one nit, otherwise lgtm

**Comment by dixudx** [comment]
@juanvallejo Updated. PTAL. Thanks.

**Comment by dixudx** [comment]
@juanvallejo @php-coder Updated. PTAL. Thanks.

**Comment by dixudx** [comment]
ping @smarterclayton @kubernetes/sig-cli-maintainers for approval. Thanks.

**Review by soltysh — APPROVED** [review]
/lgtm

**Pull request merged**